docs/source/developer/how_to_code.rst

```diff
@@ -0,0 +1,69 @@
+.. Describes how to do code
+
+.. _HowToCode:
+
+How to Code
+===========
+
+Before you start contributing in this project please read and accept our :ref:`C4`.
+
+Before you start coding always remember the *Zen of Python* by Tim Peters. If you don't know it just
+type :code:`import this` in an interactive Python interpreter.
+
+We strongly encourage you to use any Linux environment to code. The project was never tested on 
+a Windows system.
+
+
+Package Management
+------------------
+
+Use `Poetry <https://python-poetry.org/>`_ for package management.
+
+
+Code Style
+----------
+
+-   We try to follow the
+    `Google Python Style Guide <https://google.github.io/styleguide/pyguide.html>`_. Exceptions to this
+    guideline are documented below.
+
+-   Use a formatter (we suggest `Black <https://github.com/psf/black>`_) to comply with our styling
+    guidelines.
+
+-   Use a linter (we suggest `pylint <https://github.com/pylint-dev/pylint>`_) for static code
+    analysis.
+
+**Line length**
+
+Maximum line length is 100 characters.
+
```

Comment:
... These styles are enforced by a tox pipeline running `pylint` and `black` that also runs on every pull request on GitHub.